experimental/auth-logic/provenance_build_wrapper_test.go

```diff
@@ -61,4 +63,8 @@ func TestProvenanceBuildWrapper(t *testing.T) {
 		t.Errorf("got:\n%v\nwant:\n%v\n", got, want)
 	}
 
+	// cleanup
+	if err := os.RemoveAll("/tmp/release"); err != nil {
```

Comment:
Oh this is much better :)

Reply:
Does not seem like it. So I am using `os.TempDir` now. I wonder why it did not fail before on the CI.